File: server/src/main/java/com/uiptv/model/AccountInfo.java
package com.uiptv.model;

import com.uiptv.shared.BaseJson;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@EqualsAndHashCode(callSuper = false)
public class AccountInfo extends BaseJson {
    private String dbId;
    private String accountId;
    private String expireDate;
    private AccountStatus accountStatus;
    private String accountBalance;
    private String tariffName;
    private String tariffPlan;
    private String defaultTimezone;
    private String profileJson;
    private String passHash;
    private String parentPasswordHash;
    private String passwordHash;
    private String settingsPasswordHash;
    private String accountPagePasswordHash;
    private String allowedStbTypesJson;
    private String allowedStbTypesForLocalRecordingJson;
    private String preferredStbType;
}

File: server/src/main/kotlin/com/uiptv/model/AccountInfo.kt
package com.uiptv.model

import com.uiptv.shared.BaseJson

data class AccountInfo @JvmOverloads constructor(
    var dbId: String? = null,
    var accountId: String? = null,
    var expireDate: String? = null,
    var accountStatus: AccountStatus? = null,
    var accountBalance: String? = null,
    var tariffName: String? = null,
    var tariffPlan: String? = null,
    var defaultTimezone: String? = null,
    var profileJson: String? = null,
    var passHash: String? = null,
    var parentPasswordHash: String? = null,
    var passwordHash: String? = null,
    var settingsPasswordHash: String? = null,
    var accountPagePasswordHash: String? = null,
    var allowedStbTypesJson: String? = null,
    var allowedStbTypesForLocalRecordingJson: String? = null,
    var preferredStbType: String? = null
) : BaseJson()
